Source/Scene/SceneFramebuffer.js

```diff
@@ -0,0 +1,143 @@
+define([
+        '../Core/Color',
+        '../Core/defined',
+        '../Core/destroyObject',
+        '../Core/PixelFormat',
+        '../Renderer/ClearCommand',
+        '../Renderer/Framebuffer',
+        '../Renderer/PixelDatatype',
+        '../Renderer/Renderbuffer',
+        '../Renderer/RenderbufferFormat',
+        '../Renderer/RenderState',
+        '../Renderer/Sampler',
+        '../Renderer/Texture',
+        '../Renderer/TextureMagnificationFilter',
+        '../Renderer/TextureMinificationFilter',
+        '../Renderer/TextureWrap'
+    ], function(
+        Color,
+        defined,
+        destroyObject,
+        PixelFormat,
+        ClearCommand,
+        Framebuffer,
+        PixelDatatype,
+        Renderbuffer,
+        RenderbufferFormat,
+        RenderState,
+        Sampler,
+        Texture,
+        TextureMagnificationFilter,
+        TextureMinificationFilter,
+        TextureWrap) {
+    'use strict';
+
+    /**
+     * @private
+     */
+    function SceneFramebuffer() {
+        this._colorTexture = undefined;
+        this._depthStencilTexture = undefined;
+        this._depthStencilRenderbuffer = undefined;
+        this._fbo = undefined;
```

Comment:
Do we use the term FBO anywhere else in Cesium?  I think this would usually be `_frameBuffer`.